src/drive/targets/mobile/README.md

```diff
@@ -127,40 +128,45 @@ On iOS:
 Create these folders:
 
 ```
-$ mkdir src/drive/targets/mobile/keys
-$ mkdir src/drive/targets/mobile/keys/android
-$ mkdir src/drive/targets/mobile/build
-$ mkdir src/drive/targets/mobile/build/android
+$ mkdir -p src/drive/targets/mobile/keys/android && mkdir -p src/drive/targets/mobile/build/android
 ```
 
-You must have this files:
+You must have this files in `src/drive/targets/mobile/keys/android`:
 
-- keys/android/cozy-drive-release-key.jks (and the password)
-- keys/android/key.json
+- cozy-drive-release-key.jks
+- key.json
 
-To generate a signed APK on `src/drive/targets/mobile/build/android/` and publish on Google Play:
+:warning: You must also have the password associated with `cozy-drive-release-key.jks`
+
+To generate a signed APK on `src/drive/targets/mobile/build/android/`
 
 ```
-$ npm run buildsigned:drive:android
-$ npm run publish:drive:android
+$ yarn buildsigned:drive:android
 ```
 
-Or if you want to publish the release on the beta track:
+To publish on Google Play:
+
+- Manually:
+  - Go to [Google Play Console](https://play.google.com/console) and select `Drive` app
+  - To publish a beta version: go to `Release > Testing > Internal testing`
```

Comment:
C'est pas une beta version, c'est une version de test interne. 

Et ensuite, à partir de là, c'est rare de faire le yarn publish:drive:android, tu vas plutôt aller sur le playstore et dire "Rollout to production"

Reply:
J'ai corrigé pour "beta version".

Oui pour la vraie partie mise en prod c'est encore une partie manquante étant donné que je ne l'ai pas encore fait.

Par contre c'est peut-être pas clair mais il faut choisir entre "manually" et "automatically", si tu commences dans l'un tu ne finis pas avec l'autre.